README.rst

```diff
@@ -0,0 +1,107 @@
+.. image:: https://badge.fury.io/js/%40edx%2Fedx-bootstrap.svg
+    :target: https://badge.fury.io/js/%40edx%2Fedx-bootstrap
+.. highlight:: css
+    
+=============
+edX Bootstrap
+=============
+
+This library is an extension of `Bootstrap 4 <http://getbootstrap.com>`_
+for use with edX and openEdX.
+
+Installation
+------------
+
+Using npm::
+
+    npm install --save @edx/edx-bootstrap
+
+Usage
+-----
+
+Add the following import in your project's base scss file::
+
+    @import "~@edx/edx-bootstrap/scss/edx/theme.scss";
+
+Or for Open edX users,::
+
+    @import "~@edx/edx-bootstrap/scss/open-edx/theme.scss";
+
+This includes everything you need. Bootstrap is included so you don't 
+need to add it in your project. If you need more control, see the 
+theming section.
+
+Alternatively, you can download this repo and use one of the compiled outputs:
+
+- ``dist/open-edx/theme.css`` (Open edX bootstrap, extensions, & theme)
+- ``dist/edx/theme.css`` (edX bootstrap, extensions, & theme)
+- ``dist/core/core.css`` (edX bootstrap & extensions only)
+
+Theming
+-------
+
+The content of a `theme.scss` file::
+    
+    // Optional
+    @import "my-fonts";
+    @import "my-variables";
+
+    // Required core (includes bootstrap and extensions)
+    @import "@edx/edx-boostrap/scss/core/core.scss";
+
+    // The rest of your scss
+    // ...
+
+To create a theme, make a copy of the ``scss/open-edx`` folder in your own project
+and make changes. Then include it before you include edx-bootstrap::
+
+    @import "fonts";
+    @import "variables";
+
+    // If you are creating your own theme remove
+    // the "../core/core" import with the line below.
+    @import "../core/core";
+    // @import "@edx/edx-boostrap/scss/core/core.scss";
+
+    @import "overrides";
+
+Getting Help
+------------
+If you’re having trouble, contact the arch squad on Slack:
+
+-  https://openedx.slack.com: #architecture
```

Comment:
I haven't checked but this might work? slack://channel?id=<CHANNEL-ID>&team=<TEAM-ID>